tests/system-tests/rdscore/internal/rdscorecommon/egress-ip.go

```diff
@@ -0,0 +1,705 @@
+
+	return configuredEgressIPs, nil
+}
+
+// CreateEgressIPTestDeployment create egressIP test setup to verify connectivity with the external server.
+func CreateEgressIPTestDeployment() {
+	By("Creating the EgressIP test source deployment")
+
+	err := cleanUpDeployments(APIClient)
+	Expect(err).ToNot(HaveOccurred(),
+		fmt.Sprintf("Failed to cleanup deployments: %v", err))
+
+	glog.V(100).Infof("Creating the EgressIP assigned agnhost probers in namespace %s for node %s and %s",
+		RDSCoreConfig.EgressIPNamespaceOne, RDSCoreConfig.EgressIPNodeOne, RDSCoreConfig.EgressIPNodeTwo)
+
+	proberDeploySANameOne, err := createAgnhostRBAC(APIClient, RDSCoreConfig.EgressIPNamespaceOne)
+	Expect(err).ToNot(HaveOccurred(),
+		fmt.Sprintf("Failed to create prober RBAC in namespace %s: %v", RDSCoreConfig.EgressIPNamespaceOne, err))
+
+	for _, nodeToAssign := range []string{RDSCoreConfig.EgressIPNodeOne,
+		RDSCoreConfig.EgressIPNodeTwo, RDSCoreConfig.NonEgressIPNode} {
+		err = createAgnhostDeployment(
+			APIClient,
+			RDSCoreConfig.EgressIPNamespaceOne,
+			proberDeploySANameOne,
+			nodeToAssign,
+			egressIPPodLabelsMap)
+		Expect(err).ToNot(HaveOccurred(),
+			fmt.Sprintf("Failed to create prober deployment for node %s in namespace %s: %v",
+				nodeToAssign, RDSCoreConfig.EgressIPNamespaceOne, err))
+	}
+
+	glog.V(100).Infof("Creating the EgressIP assigned agnhost probers in namespace %s for node %s",
+		RDSCoreConfig.EgressIPNamespaceTwo, RDSCoreConfig.EgressIPNodeTwo)
+
+	proberDeploySANameTwo, err := createAgnhostRBAC(APIClient, RDSCoreConfig.EgressIPNamespaceTwo)
+	Expect(err).ToNot(HaveOccurred(),
+		fmt.Sprintf("Failed to create prober RBAC in namespace %s: %v", RDSCoreConfig.EgressIPNamespaceTwo, err))
+
+	err = createAgnhostDeployment(
+		APIClient,
+		RDSCoreConfig.EgressIPNamespaceTwo,
+		proberDeploySANameTwo,
+		RDSCoreConfig.EgressIPNodeTwo,
+		egressIPPodLabelsMap)
+	Expect(err).ToNot(HaveOccurred(),
+		fmt.Sprintf("Failed to create prober deployment for node %s in namespace %s: %v",
+			RDSCoreConfig.EgressIPNodeTwo, RDSCoreConfig.EgressIPNamespaceTwo, err))
+
+	glog.V(100).Infof("Creating the non EgressIP assigned agnhost prober in namespace %s for node %s",
+		RDSCoreConfig.EgressIPNamespaceOne, RDSCoreConfig.EgressIPNodeOne)
+
+	nonEIPPodLabelsMap := map[string]string{
+		strings.Split(nonEgressIPPodLabel, "=")[0]: strings.Split(nonEgressIPPodLabel, "=")[1],
+	}
+
+	err = createAgnhostDeployment(
+		APIClient,
+		RDSCoreConfig.EgressIPNamespaceOne,
+		proberDeploySANameOne,
+		RDSCoreConfig.EgressIPNodeOne,
+		nonEIPPodLabelsMap)
+	Expect(err).ToNot(HaveOccurred(),
+		fmt.Sprintf("Failed to create prober deployment for node %s in namespace %s: %v",
+			RDSCoreConfig.EgressIPNodeOne, RDSCoreConfig.EgressIPNamespaceOne, err))
+}
+
+func verifyEgressIPConnectivityBalancedTraffic(isIPv6 bool) {
+	By("Spawning the prober pod on the EgressIP non assignable host")
```

Comment:
It doesn't look accurate - there's no pod/deployment creation but listing pods in the namespace

Reply:
yep, leftovers from the old code version; thanks!